dnf.spec

```diff
@@ -21,7 +21,7 @@ URL:		https://github.com/rpm-software-management/dnf
 # cd dnf/package
 # ./archive
 # tarball will be generated in $HOME/rpmbuild/sources/
-Source0:    http://rpm-software-management.fedorapeople.org/dnf-%{version}.tar.gz
+Source0:    https://github.com/rpm-software-management/dnf/archive/%{name}-%{version}-1.tar.gz
 BuildArch:  noarch
```

Comment:
That's the question for our release management department (joke) - we will discus it.